.env

```diff
@@ -1,7 +1,8 @@
+PASS_VERSION=0.4.0
```

Comment:
Will this have to be updated manually or does it happen automagically?

Reply:
The version must be specified by the release manager when kicking off the workflow, but the automation itself will update the value in `.env`, handled by the `update-pass-version` action